### DISCUSSION TITLE
Led on switch blink when phyisical button is pressed

### DISCUSSION THREAD

**Discussion by Murder3D** [arendst/Tasmota#13432] [Q&A] [answered]
### PROBLEM DESCRIPTION
Green led lights turn on when a button is pressed on Zemismart TB21 3-gang light switch (Zemismart TB23).

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [X] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [X] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [X] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [X] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [X] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [X] Device used (e.g., Sonoff Basic): Zemismart TB21 3-gang light switch (Zemismart TB23)
- [X] Tasmota binary firmware version number used: Tasmota 10.0.0
  - [X] Pre-compiled
  - [ ] Self-compiled
- [X] Flashing tools used: Flashed Tasmota 9.5.0 using this usb-ttl adaptor --> https://www.amazon.it/DSD-TECH-Adattatore-seriale-Compatibile/dp/B07BBPX8B8/ref=sr_1_16?adgrpid=52971163419&dchild=1&gclid=EAIaIQobChMIzv-0_KHb8wIV5AN9Ch0QDQg1EAAYASAAEgIZDvD_BwE&hvadid=255215586698&hvdev=c&hvlocphy=1008359&hvnetw=g&hvqmt=e&hvrand=4208606945465942724&hvtargid=kwd-297787991276&hydadcr=28873_1803238&keywords=ttl+converter&qid=1634811114&sr=8-16

The upgrade from Tasmota 9.5.0 to Tasmota 10.0.0 was performed via web GUI

- [X] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  12:50:00.832 CMD: Backlog Template; Module; GPIO 255
12:50:00.900 MQT: stat/Soggiorno/RESULT = {"NAME":"Zemismart TB23","GPIO":[320,320,289,290,34,33,0,0,225,224,226,0,32,0],"FLAG":0,"BASE":8}
12:50:01.112 MQT: stat/Soggiorno/RESULT = {"Module":{"0":"Zemismart TB23"}}
12:50:01.367 MQT: stat/Soggiorno/RESULT = {"GPIO0":{"320":"Led_i1"},"GPIO1":{"320":"Led_i1"},"GPIO2":{"289":"Led2"},"GPIO3":{"290":"Led3"},"GPIO4":{"34":"Button3"},"GPIO5":{"33":"Button2"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"225":"Relay2"},"GPIO13":{"224":"Relay1"},"GPIO14":{"226":"Relay3"},"GPIO15":{"0":"None"},"GPIO16":{"32":"Button1"},"GPIO17":{"0":"None"}}


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [X] Provide the output of this command: `Status 0`:
```lua
  12:50:47.011 CMD: Status 0
12:50:47.019 MQT: stat/Soggiorno/STATUS = {"Status":{"Module":0,"DeviceName":"Soggiorno","FriendlyName":["Soggiorno Button1","Soggiorno Button2","Soggiorno Button3"],"Topic":"Soggiorno","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":0,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0}}
12:50:47.037 MQT: stat/Soggiorno/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota.bin.gz","RestartReason":"Software/System restart","Uptime":"0T03:02:24","StartupUTC":"2021-10-21T07:48:23","Sleep":50,"CfgHolder":4617,"BootCount":76,"BCResetTime":"2021-05-09T09:09:50","SaveCount":3216,"SaveAddress":"F7000"}}
12:50:47.050 MQT: stat/Soggiorno/STATUS2 = {"StatusFWR":{"Version":"10.0.0(tasmota)","BuildDateTime":"2021-10-19T08:23:49","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"405/699"}}
12:50:47.062 MQT: stat/Soggiorno/STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["CasaMarcoAlice",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["0000A009","2805C8000100060000005A0A000000000000","00000080","00006000","00004000"]}}
12:50:47.080 MQT: stat/Soggiorno/STATUS4 = {"StatusMEM":{"ProgramSize":612,"Free":388,"Heap":27,"ProgramFlashSize":1024,"FlashSize":1024,"FlashChipId":"1440C8","FlashFrequency":40,"FlashMode":3,"Features":["00000809","8FDAC787","04368001","000000CF","010013C0","C000F981","00004004","00001000","00000020"],"Drivers":"1,2,3,4,5,6,7,8,9,10,12,16,18,19,20,21,22,24,26,27,29,30,35,37,45","Sensors":"1,2,3,4,5,6"}}
12:50:47.095 MQT: stat/Soggiorno/STATUS5 = {"StatusNET":{"Hostname":"Soggiorno","IPAddress":"192.168.1.100","Gateway":"192.168.1.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.1.1","DNSServer2":"0.0.0.0","Mac":"XX:XX:XX:XX:XX:XX","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0}}
12:50:47.106 MQT: stat/Soggiorno/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.1.11","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_50294E","MqttUser":"mosquitto","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
12:50:47.121 MQT: stat/Soggiorno/STATUS7 = {"StatusTIM":{"UTC":"2021-10-21T10:50:47","Local":"2021-10-21T12:50:47","StartDST":"2021-03-28T02:00:00","EndDST":"2021-10-31T03:00:00","Timezone":99,"Sunrise":"08:20","Sunset":"18:48"}}
12:50:47.132 MQT: stat/Soggiorno/STATUS10 = {"StatusSNS":{"Time":"2021-10-21T12:50:47"}}
12:50:47.146 MQT: stat/Soggiorno/STATUS11 = {"StatusSTS":{"Time":"2021-10-21T12:50:47","Uptime":"0T03:02:24","UptimeSec":10944,"Heap":25,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","Wifi":{"AP":1,"SSId":"MyWifi","BSSId":"XX:XX:XX:XX:XX:XX","Channel":11,"Mode":"11n","RSSI":100,"Signal":-47,"LinkCount":1,"Downtime":"0T00:00:03"}}}


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
Upgrade from Tasmota 9.5.0 to Tasmota 10.0.0 --> Press one of the three physical button on the switch to turn on a light --> Led of all the physical buttons blink green for a second and then the led of the pressed physical button turns orange to indicate that that light is "on".

### EXPECTED BEHAVIOUR
Press one of the three phyisical button on the switch to turn on a light --> Led of only the pressed physical button turns orange to indicate that that light is "on".

### SCREENSHOTS

![Cattura](https://user-images.githubusercontent.com/68924586/138262706-e3127bbf-f778-452c-bba0-441393169514.PNG)

https://user-images.githubusercontent.com/68924586/138262720-1d19fc51-cac9-4a26-be57-323a53e37a77.mov

### ADDITIONAL CONTEXT
Basically I should not see those green leds turn on when a button is pressed.

This is the light switch I am using: **Zemismart TB21** in **3 gang** configuration (**Zemismart TB23**) --> https://www.zemismart.com/products/zemismart-tb21-smart-wifi-luxury-wall-light-switch-1-2-3-gangs-compatible-with-smart-life-app-alexa-google-home-voice-control

Here's the template I am using:

{"NAME":"Zemismart TB23","GPIO":[320,320,289,290,34,33,0,0,225,224,226,0,32,0],"FLAG":0,"BASE":8}

Sorry for any mistakes, English is not my first language and I am quite new to Tasmota troubleshooting.

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by sfromis** [comment, marked as answer]
You have a small config error, two Led_i pins with number 1. Configuring 4 Led pins should be number 1-4, irrespective of it being Led or Led_i. Of course, config details depends on what you want to use the leds for. Not sure how much this changes things, but first be sure to not have errors.

**Comment by barbudor** [comment]
Also don't forget than when you don't have a LedLink, you will have only 1 single led that will report the power status of all relays.
To have individual status with Led_1=Relay_1, led_2=relay_2 etc you Must have a LedLink assigned
If you don't have a led avaliable for that, just assign to a fee unused GPIO
